pkg/utils/utils.go

```diff
@@ -7,6 +7,20 @@ import (
 	"github.com/openshift/origin/third_party/github.com/docker/libcompose/project"
 )
 
+// CheckIfFileExists checks if all files in array exists and that they are files (not directories).
+func CheckIfFileExists(files []string) error {
+	for _, filename := range files {
+		fileInfo, err := os.Stat(filename)
+		if err != nil {
+			return fmt.Errorf("main: file %q not found", filename)
```

Comment:
I saw it in some places also, but I also saw a lot of project don't doing this.

For me it seems odd to put it there, it is just unnecessary clutter for user.